src/Templates/AdminPanel/Bit.AdminPanel/Api/Controllers/DashboardController.cs

```diff
@@ -37,15 +37,13 @@ public async Task<List<ProductsCountPerCategoryDto>> GetProductsCountPerCategoty
     [HttpGet("GetProductsSalesStats")]
     public async Task<List<ProductSaleStatDto>> GetProductsSalesStats()
     {
-        Random rand = new Random();
-
-        var products = await _dbContext.Products.ToListAsync();
-
-        return products.Select(p => new ProductSaleStatDto()
+       Random rand = new Random();
+       return   await _dbContext.Products.Include(p=>p.Category).Select(p => new ProductSaleStatDto()
         {
             ProductName = p.Name,
+            CategoryColor = p.Category!.Color,
```

Comment:
I don't like `!` operator here, can't you find a better way?

Reply:
It will never be null.

Reply:
yeah, but you need to change the code in such a way that compiler knows it.
using `!` is really bad practice, you're basically bypassing the nullable feature by this.